test/test_encode.py

```diff
@@ -75,3 +83,9 @@ def test_encode_decode():
     assert tile.southI == [0]
     assert tile.eastI == [5]
     assert tile.northI == [5]
+
+    cartesian_positions = to_ecef(positions.reshape(-1, 3))
+    normals = compute_vertex_normals(cartesian_positions, triangles)
+
+    assert np.array_equal(
+        normals, np.array(tile.vLight)), 'VertexNormals incorrect'
```

Comment:
[TerrainTile.vLight](https://github.com/loicgasser/quantized-mesh-tile/blob/master/quantized_mesh_tile/terrain.py#L388-L390)
seems like the right thing to be checking here... a bit stumped as to what may be going on

Reply:
Are the values close at all? I haven't looked at quantized mesh tile much at all.